js/src/components/contact-information/phone-number-card/verify-phone-number-content.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom/extend-expect';
+
+/**
+ * Internal dependencies
+ */
+import VerifyPhoneNumberContent from './verify-phone-number-content';
+
+describe( 'VerifyPhoneNumberContent component', () => {
+	test( 'typing Enter submits enclosing form', async () => {
+		const onVerificationStateChange = jest.fn();
```

Comment:
:nail_care: As in https://github.com/woocommerce/google-listings-and-ads/pull/1100/files#r753382342 I'd add a name, but the default report makes the reference to the source which gives a quite descriptive name anyway, so feel free to ignore this comment.